packages/bpk-docs/src/pages/AlertPage/AlertPage.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import readme from 'react-native-bpk-component-alert/readme.md';
+
+import iosScreenshotDefault from 'react-native-bpk-component-alert/screenshots/ios/default.png';
+import iosCancelable from 'react-native-bpk-component-alert/screenshots/ios/cancelable.png';
+import iosThreeButton from 'react-native-bpk-component-alert/screenshots/ios/three-button.png';
+import androidScreenshotDefault from 'react-native-bpk-component-alert/screenshots/android/default.png';
+import androidCancelable from 'react-native-bpk-component-alert/screenshots/android/cancelable.png';
+import androidThreeButton from 'react-native-bpk-component-alert/screenshots/android/three-button.png';
+import androidStylized from 'react-native-bpk-component-alert/screenshots/android/stylized.png';
+
+import IntroBlurb from './../../components/neo/IntroBlurb';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import DocsPageWrapper from './../../components/neo/DocsPageWrapper';
+import Paragraph from './../../components/Paragraph';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotDefault}`,
+        altText: 'Android Default Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'cancelable',
+    title: 'Cancelable',
+    blurb: [
+      <Paragraph>
+        Android alert can optionally be marked as cancelable. Clicking outside
```

Comment:
Two small things:

1. `alert` should be plural (`alerts`).

2. I think that this should use the British English `cancellable` (with a double `L`), as opposed to the American single `L` spelling. It's a bit odd though because the prop itself uses American English (like all our code) which is a single `L`. @jamesf3rguson what do you reckon?

Reply:
I believe that's only for the title since 'cancelable' would be the exact tag that the user needs to specify.

Reply:
The prop name is `cancelable`, but the way we refer to it in prose should be `cancellable` IMO because we aren't referencing the prop directly.

For example, I think this would be correct:

> Add the `cancelable` prop to make the alert cancellable.

Because in that case we're specifically referencing the prop.

cc @jamesf3rguson what do you think?